sw/device/tests/aes_basic_enc_dec_manualmode.c

```diff
@@ -0,0 +1,226 @@
+// Copyright lowRISC contributors (OpenTitan project).
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/lib/base/mmio.h"
+#include "sw/device/lib/dif/dif_aes.h"
+#include "sw/device/lib/dif/dif_edn.h"
+#include "sw/device/lib/dif/dif_csrng.h"
+#include "sw/device/lib/runtime/log.h"
+#include "sw/device/lib/testing/aes_testutils.h"
+#include "sw/device/lib/testing/csrng_testutils.h"
+#include "sw/device/lib/testing/test_framework/check.h"
+#include "sw/device/lib/testing/test_framework/ottf_main.h"
+
+#include "hw/top_earlgrey/sw/autogen/top_earlgrey.h"
+
+OTTF_DEFINE_TEST_CONFIG();
+
+// Register Offsets based on the OpenTitan documentation
+#define AES_CTRL_SHADOWED_REG_OFFSET         0x74
+#define AES_CTRL_AUX_SHADOWED_REG_OFFSET     0x78
+#define AES_CTRL_AUX_REGWEN_REG_OFFSET       0x7C
+#define AES_TRIGGER_REG_OFFSET               0x80
+#define AES_STATUS_REG_OFFSET                0x84
+#define AES_CTRL_AUX_SHADOWED_KEY_TOUCH_FORCES_RESEED_BIT  0
+#define AES_CTRL_AUX_SHADOWED_FORCE_MASKS_BIT              1
```

Comment:
Please remove these registers definitions. All access to registers must be done in dif libraries. See dif_aes.c for more details.